compat/src/suspense.d.ts

```diff
@@ -10,4 +10,4 @@ export interface SuspenseProps {
   fallback: preact.ComponentChildren;
 }
 
-export class Suspense extends Component<SuspenseProps> {}
+export abstract class Suspense extends Component<SuspenseProps> {}
```

Comment:
What's the reason for this being abstract? Just to avoid `new Suspense`?

Reply:
Close, it's because we declared `Component` as abstract and it expects classes that extend it to declare a render method. Because the definition didn't have one this failed. Maybe the better way is just to declare the render method here.

Reply:
alternatively `export const Suspense: Component<SuspenseProps>;`, but not sure if that would be idiomatic typescript

Reply:
Declaring render is probably the best solution. Prevents people from thinking its extendable.

Reply:
`render` should pull from the same typings as the abstract:
```
render(props?: RenderableProps<P>, state?: Readonly<S>, context?: any): ComponentChild;
```

Reply:
Imho the typing should be like this:
```
import { Component, RenderableProps, ComponentChild } from '../../src';
...
export class Suspense extends Component<SuspenseProps> {
    render(props: RenderableProps<SuspenseProps>, state: Readonly<{}>): ComponentChild;
}
```
Because in file `suspense.js` there is implementation accepting props and state
```
Suspense.prototype.render = function(props, state) {
	return state._parkedChildren ? props.fallback : props.children;
};
```